.github/workflows/linting-and-tests.yml

```diff
@@ -262,14 +262,11 @@ jobs:
           pytest -x
 
   end-to-end-tests:
-    # TODO: reenable this job once https://github.com/grafana/oncall/issues/1692 is fixed
-    if: ${{ false }}
     runs-on: ubuntu-latest
     name: "End to end tests - Grafana: ${{ matrix.grafana-image-tag }}"
     strategy:
       matrix:
         grafana-image-tag:
-          - 8.5.22
```

Comment:
OnCall does not work on this version and we may want to consider updating the dependencies object in the [plugin.json](https://github.com/grafana/oncall/blob/dev/grafana-plugin/src/plugin.json#L622-L625) to reflect this:
<img width="1222" alt="Screenshot 2023-05-17 at 19 25 02" src="https://github.com/grafana/oncall/assets/9406895/b7272d8c-071b-4cc5-9420-613bb3b5e205">